exercises/binary_converter.cpp

```diff
@@ -5,3 +5,24 @@
   Insert first number: 8
   The binary number is: 1000
 */
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+
+  int num;
+  cout << "Insert number: ";
+  cin >> num;
+
+  int temp = num;
+
+  while (temp > 0)
+  {
+    cout << temp % 2;
+    temp = temp / 2;
+  }
```

Comment:
estrai questa implementazione in una funzione a parte fuori dal main

nota: manca return 0